test/deterministic-deployment.ts

```diff
@@ -26,7 +26,10 @@ export async function setupDeployer(ethSource: Signer) {
   await ethers.provider.send("eth_sendRawTransaction", [
     DEPLOYMENT_TRANSACTION,
   ]);
-  if ((await ethers.provider.getCode(DEPLOYER_CONTRACT)).length === 0) {
+  if (
+    utils.arrayify(await ethers.provider.getCode(DEPLOYER_CONTRACT)).length ===
+    0
+  ) {
```

Comment:
This change fixes a bug I noticed while looking at this code: before it would be checking that the _string_ containing the code has length 0, which is always false since it has at least size 2: "0x".